lib/futbol_data.rb

```diff
@@ -0,0 +1,8 @@
+class FutbolData
+  attr_reader :teams_data, :game_teams_data
```

Comment:
let's check if we need attr_readers and where (which classes need them, if any)

Reply:
merging for now and we can play with cleaning this up and push the update if not needed